src/app/view/components/dot-toolbar/dot-toolbar.component.ts

```diff
@@ -1,24 +1,30 @@
 import { Component, ViewEncapsulation, Output, EventEmitter, Input } from '@angular/core';
 import { SiteService, Site } from 'dotcms-js/dotcms-js';
 import { IframeOverlayService } from '../_common/iframe/service/iframe-overlay.service';
+import { DotEventsService } from '../../../api/services/dot-events.service';
 
 @Component({
     encapsulation: ViewEncapsulation.None,
-    selector: 'toolbar-component',
-    styleUrls: ['./toolbar.component.scss'],
-    templateUrl: './toolbar.component.html'
+    selector: 'dot-toolbar-component',
+    styleUrls: ['./dot-toolbar.component.scss'],
+    templateUrl: './dot-toolbar.component.html'
 })
 export class ToolbarComponent {
     @Input() collapsed;
     @Output() mainButtonClick: EventEmitter<MouseEvent> = new EventEmitter();
 
-    constructor(public iframeOverlayService: IframeOverlayService, private siteService: SiteService) {}
+    constructor(
+        public iframeOverlayService: IframeOverlayService,
+        private siteService: SiteService,
+        private dotEventsService: DotEventsService
+    ) {}
 
     siteChange(site: Site): void {
         this.siteService.switchSite(site);
     }
 
     handleMainButtonClick($event): void {
         this.mainButtonClick.emit($event);
+        this.dotEventsService.notify({name: 'dot-toolbar-toggle'});
```

Comment:
This should go here: https://github.com/dotCMS/core-web/blob/master/src/app/view/components/main-legacy/main-legacy.component.ts#L29 because that's the place where we do toggle the side navigation... and the event should be named like: ```dot-side-nav-toogle``` because it's related with show/hide the side nav, no matter which button we toogle it with or where is located.